packages/api/src/controllers/playback.ts

```diff
@@ -19,33 +22,56 @@ type PlaybackInfo = {
     playbackPolicy?: Asset["playbackPolicy"] | Stream["playbackPolicy"];
     source: {
       // the only supported format is HLS for now
-      hrn: "HLS (TS)";
-      type: "html5/application/vnd.apple.mpegurl";
+      hrn: "HLS (TS)" | "MP4";
+      type: "html5/application/vnd.apple.mpegurl" | "html5/video/mp4";
       url: string;
+      rendition?: string;
```

Comment:
Just for my understanding - what does this field correspond to?

Reply:
This is supposed to be the rendition of the specific output, for example `360p`, however we are changing it today to `size`, as we don't have the probe data of the file yet coming back from catalyst. After we get it, we can provide additional metadata such as `bitrate` for the player

Reply:
Yeah, IMO size is a bit arbitrary to the player, since it'll have no concept of how size corresponds to the actual video. Ideally resolution details are included in the response long-term, since we're choosing a singular mp4 resolution to render since it's not streaming.

Reply:
I totally agree. Currently you can understand the resolution from the name (e.g. https://asset-cdn.lp-playback.monster/hls/e02eyumal0sigr24/static360p0.mp4), but I opened this on the api client https://github.com/livepeer/go-api-client/pull/27 which can be expanded later with the additional probe data when ready. For now only the size is included.